TOOLS.rst

```diff
@@ -0,0 +1,201 @@
+This file explains the development tool chain around ixdat
+
+Tools used
+==========
+
+The following is a list of specific tools used during ixdat
+development:
+
+* **pytest** is used for running tests
+* **flake8** is used for linting
+* **sphinx** is used to build documentation
+
+The following is a list of "tool and commands runners":
+   
+* **invoke** is used during development, to run tools and other
+  pre-configured maintenance tasks inside the existing development
+  environment
+* **tox** is used as a tool to run all **QA** tools and commands
```

Comment:
"Quality Asssureance (**QA**)", for us damn academics who don't know what QA is :wink:

Reply:
Ah dang, I though I defined it at first use :confused:  ... did for CI though though, maybe that's why I'm confused.

Fixed.